live/src/js/table/Cell.js

```diff
@@ -306,13 +306,21 @@ class Cell extends React.Component {
 	};
 
 	indexCurrentData(nextData) {
-		feed.indexData({
-			type: this.props._type,
-			id: this.props._id,
+		const { _type, _id, columnName, row, _mapping } = this.props;
+
+		const data = {
+			type: _type,
+			id: _id,
 			body: {
-				[this.props.columnName]: nextData
+				[columnName]: nextData
 			}
-		}, 'updateCell', res => this.handleErrorMsg(res));
+		};
+
+		if (_mapping && _mapping._parent && _mapping._routing && _mapping._routing.required) {
+			data.parent = row[_mapping._routing.path];
```

Comment:
@bstst I missed this earlier, but it seems ES has completely removed support for adding `path` inside `_routing` starting v2. See https://github.com/elastic/elasticsearch/issues/6730#issuecomment-64780914.

And also this https://stackoverflow.com/questions/36123647/elasticsearch-routing-on-specific-field.

In which version of ES are you able to set the `_routing.path`?

I still want to merge this, so perhaps we can add a check to ensure that `_routing.path` is present.